CMakeLists.txt

```diff
@@ -1088,15 +1088,15 @@ if(HPX_WITH_CUDA AND NOT HPX_WITH_CUDA_CLANG)
     hpx_library_dir(${CUDA_TOOLKIT_ROOT_DIR}/lib/x64)
     link_directories(${CUDA_TOOLKIT_ROOT_DIR}/lib/x64)
     set(CUDA_NVCC_FLAGS_DEBUG ${CUDA_NVCC_FLAGS_DEBUG};-D_DEBUG;-O0;-g;-G;-Xcompiler=-MDd;-Xcompiler=-Od;-Xcompiler=-Zi;-Xcompiler=-bigobj)
-    set(CUDA_NVCC_FLAGS_RELWITHDEBINFO ${CUDA_NVCC_FLAGS_RELWITHDEBINFO};-DNDEBUG;-O2;-g;-Xcompiler=-MD,-O2,-Zi;-Xcompiler=-bigobj)
+    set(CUDA_NVCC_FLAGS_RELWITHDEBINFO ${CUDA_NVCC_FLAGS_RELWITHDEBINFO};-DNDEBUG;-O2;-g;--verbose;-Xcompiler=-MD,-O2,-Zi;-Xcompiler=-bigobj;--keep)
```

Comment:
I don't see any other way as nvcc is not picking up the flags set through cmake generator settings. I had to comment out this for instance: https://github.com/STEllAR-GROUP/hpx/blob/3f87b2da11b02ff327d234adf5634d730a695bda/cmake/HPX_AddCompileFlag.cmake#L33.